### PR TITLE
Update ReadTools formula to 1.2.0

### PR DESCRIPTION
**Pull request by magicDGS**

### CODE REVIEW

File: Formula/readtools.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Readtools < Formula
-  desc "Standardizing sources for sequencing data"
+  desc "A Universal Toolkit for Handling Sequence Data from Different Sequencing Platforms"
```
Comment (magicDGS) [inline Formula/readtools.rb]: Note that I changed the description for the new repository description (matching the manuscript).

Comment (robmaz) [reply Formula/readtools.rb]: It is too long, though, with its 83 characters. brew audit also does not like beginning with indefinite articles and discourages filler terms like "framework", "program", and, I guess, also "toolkit", with little meaning beyond what is already clear from the context. I think "Handling Sequence Data from Different Sequencing Platforms" might be a description in the homebrew spirit, but is a bit boring. Maybe "Universal Toolkit for Handling Sequence Data from Different Platforms" is close enough?

Comment (magicDGS) [reply Formula/readtools.rb]: I will go for the hombrew spirit! Nothing better than following conventions for a program that it is supposed to be for "standardizing" formats...hahaha. Next commit have `Handling Sequence Data from Different Sequencing Platforms` instead.

Thanks for the feedback!

**Comment by magicDGS** [comment]
Now it should be ready!

**Pull request merged**